src/reducers/counter.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+
+import { Actions } from '../actions/counter';
+
+export const CounterReducer = (state = { count: 10 }, action) => { 
```

Comment:
Break out the initial state into const. This makes it easier to see what the initial state is and change it in the future. Here is how I do it in Live: https://github.com/showclix/Live-React-Native/blob/dev/src/reducers/device.ts#L7.